projects/libs/core-library-typescript/src/block-storage/block-storage.ts

```diff
@@ -1,9 +1,14 @@
 import * as path from "path";
 
 import { LevelDBDriver } from "./leveldb-driver";
-import { BlockUtils , logger, types, JsonBuffer } from "../common-library";
+import { BlockUtils, logger, types, JsonBuffer } from "../common-library";
+import { STARTUP_STATUS, StartupStatus } from "../common-library/startup-status";
+import { StartupCheck } from "../common-library/startup-check";
+
+export class BlockStorage implements StartupCheck {
+
+  private SERVICE_NAME = "block";
```

Comment:
we have block storage and block builder logic units, maybe be more specific here

Reply:
also maybe make it a 'readonly'